packages/ddp-client/client/client_convenience.js

```diff
@@ -42,6 +43,9 @@ Meteor.connection = DDP.connect(ddpUrl, {
   onDDPVersionNegotiationFailure: onDDPVersionNegotiationFailure
 });
 
+// Makes sure Meteor.connect is already created before injecting the helpers
+loadAsyncStubHelpers();
```

Comment:
I'm not very familiar with this part of Meteor. Should these also be used for DDP clients created on the server? I would expect method stubs run on the server to have at least some of the same problems.

Reply:
I think we only need it on the server. Let's keep this way for now and change it in the future if needed.